client/signup/main.jsx

```diff
@@ -500,7 +490,7 @@ class Signup extends React.Component {
 						signupDependencies={ this.props.signupDependencies }
 						stepSectionName={ this.props.stepSectionName }
 						positionInFlow={ this.positionInFlow() }
-						hideFreePlan={ hideFreePlan }
```

Comment:
We can't just make it `true`. This will be broken in the following cases: https://github.com/Automattic/wp-calypso/pull/22430 https://github.com/Automattic/wp-calypso/pull/19390